data/event-schemas/schema/m.room.power_levels.yaml

```diff
@@ -75,9 +65,6 @@ properties:
         description: |-
             The default level required to send state events. Can be overridden
             by the `events` key. Defaults to 50 if unspecified.
-
-            **Note**: When there is no `m.room.power_levels` event in the room, this defaults
-            to 0. See description for more information.
```

Comment:
I probably should have done it in my original PR, but while you're here it'd be good to have a similar note for the "creator gets 100" clause on `users_default` (effectively).

Reply:
Added a note in 4a90545e3da7084415710e08aeb676c9191af1cb (and clarified the rest of the description of `users_default` too